backend/modules/professor/routers/professor_grade_export.py

```
@@ -0,0 +1,78 @@
+from common.db.models import User, StudentSimulationInstance
+from common.db.models.cohorts.cohort import Cohort, CohortStudent
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter(prefix="/professor/grades", tags=["Professor Grade Export"])
+
+
+@router.get("/cohorts/{cohort_id}/export")
+async def export_cohort_grades(
+    cohort_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.15.7)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)